build.gradle.kts

```diff
@@ -42,12 +42,12 @@ subprojects {
     dependsOn("spotlessKotlinApply")
     compilerOptions {
       jvmTarget = JvmTarget.JVM_11
+      freeCompilerArgs.add("-Xjdk-release=11")
```

Comment:
@staktrace Is there a reason why you did it this way?

I'm running into trouble with my gradle plugin now
`e: '-Xjdk-release=11' option conflicts with '-jvm-target 17'. Please remove the '-jvm-target' option`

Reply:
Yeah the gradle plugin sets the target to 17 because it needs it for SqlDelight integration. Plus the artifact is a gradle plugin not a java library.

Reply:
This here:
https://github.com/cashapp/backfila/blob/master/client-sqldelight-gradle-plugin/build.gradle.kts#L47-L55